src/main/java/org/broadinstitute/barclay/argparser/CommandLineArgumentParser.java

```diff
@@ -690,11 +690,17 @@ private void setArgument(ArgumentDefinition argumentDefinition, List<String> val
             if (argumentDefinition.isCollection) {
                 @SuppressWarnings("rawtypes")
                 final Collection c = (Collection) argumentDefinition.getFieldValue();
-                if (value == null) {
-                    //user specified this arg=null which is interpreted as empty list
-                    c.clear();
-                } else {
-                    c.add(value);
```

Comment:
There is at least one other place in this method that calls clear on a collection (line 627). We should scan the code for any others.

Reply:
Done. I added a method to catch the exception while applying a consumer (either clear/add, but in the future may be others).